App Development/readme.md

```diff
@@ -12,6 +12,7 @@ App development refers to crafting software for diverse devices, involving plann
 - [YouTube Channels](#youtube-channels)<br>
 - [YouTube Playlists](#youtube-playlists)
 - [GitHub Repositories](#github-repositories)
+- [Projects](#github-repositories)
```

Comment:
```suggestion
- [Projects](#projects)
```

Reply:
Sorry for trouble sir i resolved the issue u can merge the R @jfmartinz sir 
Than you